userspace/libscap/scap_procs.c

```diff
@@ -242,13 +284,13 @@ int32_t scap_proc_fill_info_from_stats(scap_t *handle, char* procdirname, struct
 			}
 		}
 
-		if(nfound == 10)
+		if(nfound == 13)
 		{
 			break;
 		}
 	}
 
-	ASSERT(nfound == 10 || nfound == 7 || nfound == 6);
+	ASSERT(nfound == 13 || nfound == 7 || nfound == 6);
```

Comment:
I think `nfound == 7 || nfound == 6` is a really weird check; did you understand by chance why is that check there?
Perhaps we could use a bitmask flag here, something like
```
nfound |= TINFO_UID_F;
nfound |= TINFO_CAPEFF_F;

...

if (nfound == TINFO_MAX_F || nfound == TINFO_xx_F || ...)
```

(given that i do not know that `7` and `6` means in the above check! :D

Reply:
Yes, it is indeed a really weird check, but I assumed that only touching the `nfound == 10` would not break the other two checks. I suppose that in old kernels, not all the fields that are nowadays in `/proc/<pid>/status` were there and fields got into that file incrementally, that's why I would stick with that check!

Reply:
That makes sense; still, i'd like to have some sort of documentation on top of that (just a comment is fine).  
I mean, my proposed code would be self-documenting but i am ok with leaving it as it is!